eth_tester/main.py

```diff
@@ -254,6 +254,11 @@ def get_code(self, account, block_number="latest"):
         code = self.normalizer.normalize_outbound_code(raw_code)
         return code
 
+    def set_code(self, account, code):
+        self.validator.validate_inbound_account(account)
+        raw_account = self.normalizer.normalize_inbound_account(account)
+        self.backend.set_code(raw_account, code)
```

Comment:
i think this might be too irregular of a state to work, i have tried `.persist()` but cant get things to really persist